spec/models/course_spec.rb

```diff
@@ -741,4 +741,28 @@
       end
     end
   end
+
+  describe '#syncable?' do
+    let(:subjects) { [build(:subject, subject_name: "primary")] }
+    let(:site_status) { build(:site_status, :findable) }
+
+    subject { create(:course, subjects: subjects, site_statuses: [site_status]) }
+
+    context 'course which is valid' do
+      let(:subject) { create(:course, subjects: subjects, site_statuses: [site_status]) }
```

Comment:
You don't need this because of the `subject` above. You don't need to `let(:subject) {...}`, this is enough: `subject {...}`.

Reply:
In fact, I'm not sure you even really need the `context` block here, since you could just say that by default, the course is valid, which seems like a reasonable assumption.

Reply:
Have actioned.